eq-author/src/components/Toasts/ToastContainer/index.js

```diff
@@ -38,15 +30,15 @@ const ToastWrapper = styled.div`
   justify-content: center;
 `;
 
-export const ToastArea = ({ toasts, dismissToast }) => {
+export const ToastArea = ({ toasts, onDismissToast }) => {
   return (
     <ToastWrapper>
       <ToastOuterContainer>
         <ToastInnerContainer>
           <ToastList>
-            {map(toasts, (toast, id) => (
-              <Toast key={id} id={id} timeout={5000} onClose={dismissToast}>
-                <StyledDiv>{toast.message}</StyledDiv>
+            {toasts.map(({ id, message }) => (
+              <Toast key={id} id={id} timeout={5000} onClose={onDismissToast}>
```

Comment:
Im not sure on what the conventions are, but I find it a little confusing not passing a `handleFunction` to an `onClose`?

Reply:
We don't need the handle as is it just passing the handler down. So it would seem weird to rename it in the component.

Reply:
But could you change `<ToastContainer toasts={toasts} onDismissToast={handleDismissToast} />` to `<ToastContainer toasts={toasts} handleDismissToast={handleDismissToast} />`? and then use that?

Reply:
As discussed leaving as is